public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx

```diff
@@ -327,7 +327,7 @@ export const MetricEncyclopediaModal = (props: Props) => {
         setMetrics(metrics);
         setFilteredMetricCount(metrics.length);
         setIsLoading(false);
-      }, 300),
+      }, datasource.getDebounceTimeInMilliseconds() ?? 350),
```

Comment:
It looks like getDebounceTimeInMilliseconds is returning 350 by default. Do we need `??` here?

Reply:
DOUBLE TAP. Yeah I'll fix